BE/OwnershipAndControl/CorporateControl.rdf

```diff
@@ -264,55 +269,62 @@
 		<rdfs:label>has majority controlling interest party</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-be-le-cb;StockCorporation"/>
-		<rdfs:range rdf:resource="&fibo-be-oac-cctl;OverFiftyPercentControllingInterestCompany"/>
-		<skos:definition>relates a company to an organization that has a majority ownership and control over it, i.e., its parent organization, if there is one</skos:definition>
-		<skos:editorialNote>This is defined as company or other Formal Organization which owns a controlling stake of greater than 50 percent (50 percent plus one voting share or above) in this company.</skos:editorialNote>
+		<rdfs:domain rdf:resource="&fibo-be-le-lp;LegalEntity"/>
+		<rdfs:range rdf:resource="&fibo-be-oac-cctl;OverFiftyPercentControllingInterestParty"/>
+		<skos:definition>relates a legal entity to a party that owns a controlling stake of greater than 50 percent (50 percent plus one voting share or above)</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;hasMajorityOwnedSubsidiary">
 		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-cctl;hasSubsidiary"/>
 		<rdfs:label>has majority owned subsidiary</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-be-le-cb;StockCorporation"/>
+		<rdfs:domain rdf:resource="&fibo-be-le-lp;LegalEntity"/>
 		<rdfs:range rdf:resource="&fibo-be-oac-cctl;WhollyOwnedSubsidiary"/>
 		<skos:definition>relates a company to one of its majority-owned subsidiaries, i.e., a subsidiary of which it owns more than 50 percent (50 percent plus one share) of the outstanding shares</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;hasSubsidiary">
 		<rdfs:label>has subsidiary</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-be-le-cb;StockCorporation"/>
+		<rdfs:domain rdf:resource="&fibo-be-le-lp;LegalEntity"/>
 		<rdfs:range rdf:resource="&fibo-be-oac-cctl;Subsidiary"/>
-		<skos:definition>relates a company to one of its subsidiaries, that is an affiliae controlled by the company directly, or indirectly through one or more intermediaries</skos:definition>
+		<skos:definition>relates a legal entity to another organization that it controls directly, or indirectly, through one or more intermediaries</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;holdsMajorityControllingVotingRightsIn">
 		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-cctl;holdsSignificantControllingVotingRightsIn"/>
 		<rdfs:label>holds majority controlling voting rights in</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-be-le-lp;LegalEntity"/>
-		<skos:definition>relates a legal person to a company in which the party holds fifty percent or more of the controlling voting rights</skos:definition>
+		<skos:definition>relates a party to a legal entity in which it holds fifty percent or more of the controlling voting rights</skos:definition>
```

Comment:
Circular - in particular "controlling voting rights" is undefined. How related to the number of votes on issued(?) shares

Reply:
I've eliminated the word controlling - voting rights is sufficient.  One can hold other people's voting rights contractually even if you don't own the shares - I think Mark Zuckerberg has this capability based on news articles.   The term 'controlling' is from the original ontology that Mike created - so we might revisit that under another issue, again once the lattice is in place and we can come back to these properties to re-configure those that are important and eliminate those that are not needed anymore.